src/libraries/Microsoft.PowerFx.Interpreter/Environment/PowerFxConfigExtensions.cs

```diff
@@ -39,7 +39,11 @@ public static void EnableMutationFunctions(this SymbolTable symbolTable)
         {
             symbolTable.AddFunction(new RecalcEngineSetFunction());
             symbolTable.AddFunction(new CollectFunction());
-            symbolTable.AddFunction(new PatchFunction());
+            symbolTable.AddFunction(new PatchImpl());
+            symbolTable.AddFunction(new PatchSingleRecordImpl());
+            symbolTable.AddFunction(new PatchAggregateImpl());
+            symbolTable.AddFunction(new PatchAggregateSingleTableImpl());
+            symbolTable.AddFunction(new PatchRecordImpl());            
```

Comment:
>symbolTable.AddFunction(new PatchRecordImpl() [](http://example.com/codeflow?start=12&length=45)

This is not a mutation function - it's a pure function with no side effects. It should go to the core list.

Reply:
Moved to Core.